### ISSUE TITLE
Username/password not encrypted?

### ISSUE THREAD

**Issue by jackkamm** [captin411/ofxclient#74]
The readme states:

> Your username and password are stored encrypted however your account number, routing number, bank name, and account type are not encrypted and are visible to anyone with access to this file on your local computer.

However, when I look at my `~/ofxclient.ini`, the username and password are NOT encrypted, under the `institution.username` and `institution.password` fields.

Is this expected behavior, and the Readme incorrect? Or am I running ofxclient incorrectly? Any tips to configure/run ofxclient so that these sensitive fields are encrypted?

**Comment by jackkamm** [comment]
After I installed & configured a keyring manager on my system, ofxclient started storing the username and password encrypted, as desired.

In particular, I installed GNOME Keyring, but any keyring usable by the Python [keyring package](https://pypi.org/project/keyring/) should work.

**Comment by emin-ec** [comment]
I had a similar issue to this: I had a minimal ubuntu system with no keyring installed. When I ran `ofxclient` it stored username and password in the clear. While I think `ofxclient` is great and understand the author is busy, this is not the safest behavior. If someone more knowledgeable could put in exception to prevent storing a password in the clear, that would be good.

The work-around I used was to `pip install keyring` and `pip install keyring.cryptfile` and then follow instructions at https://pypi.org/project/keyrings.cryptfile/ to create a keyring that ofxclient could use.

Failing that, maybe we should re-open this ticket?
